packages/astro/src/default/components/ResizablePanel.astro

```diff
@@ -208,89 +233,3 @@ if (!hasSidePanel) {
 
   customElements.define('resizable-panel', ResizablePanel);
 </script>
-
-<style>
```

Comment:
I replaced the inline style from this component with unocss classes as it makes it easier to add breakpoints for mobile.